### PR TITLE
Localisation support

### PR DESCRIPTION
**Pull request by tinygrox**
Make mod translatable! Chinese and English languages are now supported.

### CODE REVIEW

**Comment by JonnyOThan** [comment]
Wow this is great!!! A couple notes:
-I don't think it's useful to have a comment with the string contents at each usage site when it's usually just the same as the variable name.  This isn't a big deal though, and I can remove them later if you don't feel like updating this.
-There are some user-facing strings in GameData/FreeIva/Patches/AutoPassThrough.cfg and GameData/FreeIva/UniversalStorage.cfg.  Look for `partInfo` and `FreeIva_AutoPassThrough_Error`.  Do you think you could localize those?

Thank you!!

**Comment by tinygrox** [comment]
took a while but it's all Done!

File: GameData/FreeIva/Localization/en-us.cfg [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        // Part Info texts on the left in VAB/SPH
+        #FreeIVA_PartInfo_CanTraverse = This part can be traversed in IVA
+        #FreeIVA_PartInfo_NotBlockHatch = This part will not block EVA hatches
+        
+        // For other Translators: DO NOT DELETE the "#" symbol -- this is for MM patches in AutoPassThrough.cfg 
+        // %partInfo = #FreeIVA_PartInfo_CanTraverse_Penalty
+        // equals
+        // %partInfo = #This part can be traversible in IVA with a penalty of $/FreeIva_AutoPassThrough_VolumePercent$% of its tank volume.
+        #FreeIVA_PartInfo_CanTraverse_Penalty = #This part can be traversible in IVA with a penalty of $/FreeIva_AutoPassThrough_VolumePercent$% of its tank volume.
```
Comment (JonnyOThan) [inline GameData/FreeIva/Localization/en-us.cfg:50]: Does this actually work?  That's really surprising, but I'll try and figure something out...

Comment (tinygrox) [reply GameData/FreeIva/Localization/en-us.cfg:50]: Yeah it works, and well 😄

**Pull request merged**